acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/jsonrpc/HttpServiceLoginAcceptanceTest.java

```diff
@@ -22,12 +25,17 @@
 import org.junit.Test;
 
 public class HttpServiceLoginAcceptanceTest extends AcceptanceTestBase {
+  private Cluster authenticatedCluster;
   private Node node;
 
   @Before
   public void setUp() throws IOException, URISyntaxException {
+    final ClusterConfiguration clusterConfiguration =
+        new ClusterConfigurationBuilder().setAwaitPeerDiscovery(false).build();
```

Comment:
With the await peer discovery being off is there still something that is awaiting the node being ready? Want to avoid the login call further down occurring before the service has finished starting in the node.

Reply:
there is nothing awaiting the node being ready. If you can't do any JSON RPC calls, is there another way to ask the node if it's ready?

Reply:
^ if this is a problem, it's a problem for a few tests

Reply:
If we want to test that the services are up (JSON-RPC service), can't we send a request to the 'login' route and expect any response?

Reply:
that's exactly what the test does. I think Chris is asking is there a chance that the login call happens before the node has started?

Reply:
What I meant is that we could have a check (prior to the test) if the JSON-RPC service is responding. Currently, we use the admin_peers I guess, we could have an alternative (for when we disable the discovery check) that would send a request to the `login` route and expect it to be handled (it doesn't need to be a successful login, just need the request to be "received" by the node).

Reply:
gotcha. awaitLogin()